synapse/lib/storm.py

```diff
@@ -388,3 +394,167 @@ def runStormCmd(self, snap, genr):
             node = snap.getNodeByBuid(buid)
             if node:
                 yield node, node.initPath()
+
+class RefsCmd(Cmd):
+    '''
+    Get nodes adjacent to inbound nodes.
+    '''
+    name = 'refs'
+    def getArgParser(self):
+        pars = Cmd.getArgParser(self)
+        pars.add_argument('-d', '--degrees', type=int, default=1, action='store',
+                          help='Number of degrees to walk from the source node.')
+        pars.add_argument('--traverse-edge', default=False, action='store_true',
+                          help='Traverse Edge type nodes, if encountered, to '
+                               'the opposite side of them, if the opposite '
+                               'side has not yet been encountered.')
+        pars.add_argument('--join', default=False, action='store_true',
+                          help='Include source nodes in the output of the refs command.')
+        pars.add_argument('--omit-traversal-form', action='append', default=[], type=str,
```

Comment:
Some of these arguments are pretty long - i'm open to changing them and providing short (single character) alternatives.

Reply:
+1 to having a short form and long form.